### DISCUSSION TITLE
Oracle TimesTen connection through HammerDB

### DISCUSSION THREAD

**Discussion by vikram-umachagi** [TPC-Council/HammerDB#576] [General]
Need help connecting to TimesTen DB using HammerDB. I am able to connect to the timesten db using the sys.odbc.ini client config files using TCP/TLS. When we try create schema and provide the similar options its failing with "ORA-12154 TNS: unable to resolve connect identifier". 
The user created for the instance had been given admin privilege.
Any document or blog on how to connect to TimesTen DB using HammerDB?

**Comment by sm-shaw** [comment]
The most recent update I had was that HammerDB for TimesTen was still working out of the box in 2019 and nothing has changed on the HammerDB side since then.  The troubleshooting guide is here: https://docs.oracle.com/database/timesten-18.1/TTTRB/TTTRB.pdf
in particular, you should refer to the section "If you receive error ORA-12514".  Once you can connect then try HammerDB again.
